IFC4x3/Properties/f/FiberType_33hW6SI9L88u4r9StkBaiz/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FiberType_33hW6SI9L88u4r9StkBaiz" Name="FiberType" UniqueId="c3ae019c-4895-4823-8135-25cdee2e4b3d" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
Property with same name above.  Is it the same or resolve name.

Reply:
Solved! commit : 1502e35a1872570498aaf3ca276776848d74b6dd